changelog/DIP25-is-default.dd

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
I'm not sure where the doc system can handle entries without a title.

Reply:
It can't. Needs to be a title followed by a blank line followed by a description, I learnt that the hard way.

Reply:
I know, I wrote it. I was just trying to be gentle.

Reply:
and I'm removing any ambiguity because I've had about as much fun with these things as Walter and I know I'd rather just be told what needs doing ;-)